docs/src/main/paradox/server/walkthrough.md

```diff
@@ -266,6 +267,8 @@ request metadata parameter to each service method. See the detailed chapters on
 and @ref[Maven](../buildtools/maven.md) for how to set this build option. Note that this option doesn't effect the
 generated client stubs.
 
+Notice: you need change `GreeterServiceHandlerFactory` to `GreeterServiceHandlerFactoryPowerApiHandlerFactory`
```

Comment:
I think these class names are java-specific. Could you wrap them in `@java[....]` and add the corresponding `@scala[...]` alternative?

Reply:
Sure.

Reply:
In scala, does the java class `GreeterServiceHandlerFactoryPowerApiHandlerFactory` is `GreeterServiceHandlerPowerApi`  ?